src/components/AccomodationCard.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Placeholder from 'react-bootstrap/Placeholder';
+import Rating from '@mui/material/Rating';
+import ImageIcon from '@mui/icons-material/Image';
+import AspectRatio from '@mui/joy/AspectRatio';
+import './AccomodationCard.css'
+
+type AccomodationProps = {
+    title: string;
+    location: string;
+    price: number;
+    categorization: number;
+};
+
+const AccomodationCard = (props: AccomodationProps) => {
+    const title = props.title;
+    const location = props.location;
+    const price = props.price;
+    const categorization = props.categorization;
```

Comment:
You can use object destructuring instead.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment

Reply:
Do this for all similar cases